### DISCUSSION TITLE
CTkOptionMenu  and CTkComboBox showing old style menu

### DISCUSSION THREAD

**Discussion by ghost** [TomSchimansky/CustomTkinter#2147] [General]
I have tried both CTkOptionMenu  and CTkComboBox but they show the old style menu with a frame around it rather than the new customtkinter style menu. I am importing as follows:

`from customtkinter import CTkToplevel, CTkButton, CTkLabel, CTkOptionMenu, CTkComboBox`

an using as follows:

```
        self._transactionTypeType = CTkOptionMenu(
            self._frames._frameMain,
            values=[
                "Transfers funds between two accounts",
                "Deposits funds into an account",
                "Withdraws funds from an account",
            ],
            dropdown_fg_color=COLOR,
            width=400,
            anchor="w",
        )
        self._transactionTypeType.grid(row=3, column=1, padx=(0, 10), pady=(10, 0))
```

but i get a menu with a box around rather than the one shown in the docs.
How can i solve this?

I've tried the 'remove border' approach but that doesn't work.

thanks

**Comment by Akascape** [comment]
@Barrowcroft Try this dropdown: https://github.com/Akascape/CTkScrollableDropdown

**Reply by ghost** [reply]
Thank you. Yes, I had seen that but wanted to get the basic version working correctly before trying somethings else.

**Reply by Akascape** [reply]
@Barrowcroft I don't think it will be modified, the default dropdown is actually a tk,menu, the white border around the menu is permanent in windows. Moreover, the older version of ctk used to have the same dropdown I made, but it was removed after issue with macos. But I fixed most of the issue and uploaded the new dropdown. But still due to some reason, that type of dropdown will not be implemented for now. So better switch to my dropdown if you are concerning about the looks.

**Reply by ghost** [reply]
ok ,thanks, i will give it a try
